libraries/chain/dbs_registration_committee.cpp

```diff
@@ -0,0 +1,141 @@
+{
+    FC_ASSERT(!genesis_state.registration_committee.empty(), "Registration committee must have at least one member.");
+    FC_ASSERT(SCORUM_REGISTRATION_LIMIT_COUNT_COMMITTEE_MEMBERS > 0, "Invalid ${1} value.",
+              ("1", SCORUM_REGISTRATION_LIMIT_COUNT_COMMITTEE_MEMBERS));
+
+    // check existence here to allow unit tests check input data even if object exists in DB
+    FC_ASSERT(get_committee().empty(), "Can't create more than one committee.");
+
+    const dbs_account& account_service = db().obtain_service<dbs_account>();
+
+    // create unique accout list form genesis unordered data
+    using sorted_type = std::map<account_name_type, std::reference_wrapper<const account_object>>;
+    sorted_type items;
+    for (const auto& genesis_item : genesis_state.registration_committee)
+    {
+        const account_object& accout = account_service.get_account(genesis_item);
+
+        items.insert(sorted_type::value_type(accout.name, std::cref(accout)));
+    }
+
+    // add members
+    for (const auto& item : items)
+    {
+        if (_get_member_count() > SCORUM_REGISTRATION_LIMIT_COUNT_COMMITTEE_MEMBERS)
+        {
+            wlog("Too many committee members in genesis state. More than ${1} are ignored.",
+                 ("1", SCORUM_REGISTRATION_LIMIT_COUNT_COMMITTEE_MEMBERS));
+            break;
+        }
+        const account_object& accout = item.second;
+        _add_member(accout);
+    }
+
+    if (!_get_member_count())
+    {
+        FC_ASSERT(false, "Can't initialize at least one member.");
+    }
+
+    return get_committee();
+}
+
+const registration_committee_member_object&
+dbs_registration_committee::add_member(const account_name_type& account_name)
+{
+    FC_ASSERT(!get_committee().empty(), "No committee to add memeber.");
+
+    const dbs_account& account_service = db().obtain_service<dbs_account>();
+
+    const account_object& accout = account_service.get_account(account_name);
+
+    return _add_member(accout);
+}
+
+void dbs_registration_committee::exclude_member(const account_name_type& account_name)
+{
+    FC_ASSERT(!get_committee().empty(), "No committee to exclude memeber.");
+
+    const dbs_account& account_service = db().obtain_service<dbs_account>();
+
+    const account_object& accout = account_service.get_account(account_name);
+
+    _exclude_member(accout);
+}
+
+void dbs_registration_committee::update_member_info(const registration_committee_member_object& member,
+                                                    member_info_modifier_type modifier)
+{
+    db_impl().modify(member, [&](registration_committee_member_object& m) { modifier(m); });
+}
+
+uint64_t dbs_registration_committee::_get_member_count() const
+{
+    return db_impl().get_index<registration_committee_member_index>().indicies().size();
+}
+
+bool dbs_registration_committee::_check_member_exist(const account_name_type& account_name) const
```

Comment:
Just _member_exist() name for function is more comprehensible

Reply:
I don't think sow